lessons/specifics/poolboy.md

```diff
@@ -0,0 +1,137 @@
+---
+layout: page
+title: Poolboy
+category: specifics
+order: 1
```

Comment:
I think it's a valid concern.  @ayayalar if you're open to we could just leave this PR for a bit while we think about it further.

How we could handle libraries is something I've been thinking about for awhile now and it's a tough one.  On one hand we don't want to have 10000 different lessons, many of which are for libraries that will be abandoned at some point.  On the other, there are libraries that are quite prevalent in the ecosystem and it seems worth it to have them covered. 

I'm open to thinking about this a bit further.